compose/README.md

```diff
@@ -0,0 +1,292 @@
+# compose
+
+## Index
+
+- [Information](#information)
+- [Installation](#installation)
+- [Plan](#plan)
+- [Build](#build)
+- [Run](#run)
+  - [Local](#local)
+  - [Antithesis](#antithesis)
+- [Troubleshoot](#troubleshoot)
+- [Appendix](#appendix)
+
+## Information
+
+This document describes the process of setting up a Cardano testnet from scratch. `cardano-node` can be compiled from source or downloaded as pre-compiled binary. The testnet will run inside of **Docker Containers** and will be controlled by **Docker Compose**.
+
+## Installation
+
+Docker is the only requirement to build and run your own testnets. The following instructions are tested on Debian 12.
+
+- Install dependencies via APT package manager
+
+  ```
+  sudo apt install --no-install-recommends \
+      ca-certificates \
+      curl \
+      make
+  ```
+
+- Add Docker APT repository public key
+
+  ```
+  sudo install -m 0755 -d /etc/apt/keyrings
+  sudo curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc
+  sudo chmod a+r /etc/apt/keyrings/docker.asc
+  ```
+
+- Add Docker APT repository
+
+  ```
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+  ```
+
+- Refresh APT cache
+
+  ```
+  sudo apt update
+  ```
+
+- Install Docker
+
+  ```
+  sudo apt install --no-install-recommends \
+      docker-ce \
+      docker-ce-cli \
+      containerd.io \
+      docker-buildx-plugin \
+      docker-compose-plugin
+  ```
+
+- Add your user to the `docker` group
+
+  ```
+  sudo usermod --append --groups docker $(whoami)
+  ```
+
+- Pull and run the Docker test image
+
+  ```
+  docker run hello-world
+  ```
+
+You are now ready to build your own testnet container images from source or pre-compiled binary.
+
+## Plan
+
+- Clone the `antithesis.git` Git repository
```

Comment:
Not really needed as these instructions are in the antithesis repository you're supposed to clone, I guess?